examples/chunking.rs

```diff
@@ -37,7 +38,7 @@ fn spawn_chunk(commands: &mut Commands, asset_server: &AssetServer, chunk_pos: I
         .entity(tilemap_entity)
         .insert_bundle(TilemapBundle {
             grid_size: TILE_SIZE.into(),
-            size: CHUNK_SIZE,
+            size: CHUNK_SIZE.into(),
```

Comment:
I think this might be confusing. It's best to not equate render chunks with user built tile map chunks here. 

I think what I would like to see is something like:

```rust
const CHUNK_SIZE: UVec2 = UVec2 { x: 32, y: 32 };

// Render chunk sizes are set to 4 render chunks per user specified chunk.
TilemapRenderSettings {
    chunk_size: UVec2::new(16.0, 16.0),
}
```

I'm open to suggestions here too though. 👍